### PR TITLE
Add CQL syntax highlighting

### PR DESCRIPTION
**Pull request by clohfink**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -68,24 +76,21 @@ <h1 style="display:inline">CQL Data Modeler</h1>
                 Provide some information about your table definition, and the size, population, and cluster distributions of each of your fields and the tool will give you a script you can use to benchmark your data model and a storage engine visualization.</br></br>
                 Please input your table definition for one table, keeping the Create statement, field definitions, and Primary Key definition on separate lines.</p>
                 <input style="display:block" id="keyspace" placeholder="Keyspace Name" value="test_keyspace"></input>	</br>
-                <textarea id="tableDef" data-role="none"  placeholder="Create Table Statement"  style="width:600px; height:250px">CREATE TABLE reviews_by_day
```
Comment (phact) [inline index.html]: Does the placeholder break highlighting?

File: js/sizer.js [hunk 2]
```diff
@@ -42,33 +33,14 @@ var processTableDef = function(value){
...
-	value = value.replace(dashCommentsRegex,"\n");
-	value = value.replace(slashCommentsRegex,"\n");
-	value = value.replace(extraSpaceRegex,"\n");
-	value = value.replace(emptyLinesRegex,"");
-
-    pk = $("#tableDef").val().match(pkWithCloseParen)
-	value = value.replace(pkWithCloseParen,"")
-	value = value.replace(endOfColumnRegex,",\n")
-	value = value + pk[0].trim()
-	value = value.replace(emptyLinesRegex,"");
```
Comment (phact) [inline js/sizer.js]: This still needs to happen for the regex parsing to work later on. I'm okay with keeping the cleaned up value in a variable and printing to console.log if you want to leave the text in the text area as is.

File: js/sizer.js [hunk 3]
```diff
@@ -361,7 +322,7 @@ Sum of the size of the Keys + Sum of the size of the static columns + Number of
 
 //Ugly....
 function downloadYaml(filename) {
-  var text = $("#tableDef").val();
+  var text = editor.getValue();
```
Comment (phact) [inline js/sizer.js]: I'm not sure if we would use the variable (processed version of the table) for the file output, or if we should use the table def verbatim as the user created it. I think we're better off using processed version to ensure that we have a valid yaml.

File: index.html (hunk 1 above)
Comment (clohfink) [inline index.html]: no but since it has data in it to start i didnt feel it necessary, ill put it back though in followup here

**Comment by phact** [comment]
Great!

+1 lgmt, merging

**Pull request merged**